api/api_test.go

```diff
@@ -745,8 +745,7 @@ func TestWebsocketClientDispatchEventWithoutMutatingData(t *testing.T) {
 		  "version": "",
 		  "extensionPoints": null,
 		  "localization": null,
-		  "surface": "checkout",
-		  "canAccessNetwork": false
+		  "surface": "checkout"
```

Comment:
If my feedback about `omitempty` on `Capabilities` is accepted, then we'll likely need to add `"capabilities": []` to this expected response.

Reply:
resolved with @vividviolet's suggestion and our Slack discussion.